qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/conda-store/config/conda_store_config.py

```diff
@@ -1,29 +1,43 @@
+import json
 import logging
 import os
+import tempfile
 
 import requests
 from conda_store_server import api, orm, schema
 from conda_store_server.server.auth import GenericOAuthAuthentication
-from conda_store_server.server.utils import get_conda_store
+from conda_store_server.server.dependencies import get_conda_store
 from conda_store_server.storage import S3Storage
 
+
+def conda_store_config(path="/var/lib/conda-store/config.json"):
```

Comment:
Just curious, where does this file come from?

Reply:
This is a pattern that I copied from zero-to-jupyterhub. https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/main/jupyterhub/files/hub/z2jh.py#L15. The idea is to mount the secret into the container at `/var/lib/conda-store` with a data key of `config.json` and value of a json dictionary. This allows us to pass in json data without having to templatize the files. Dask gateway does this as well now.